### PR TITLE
Add /i command

### PR DESCRIPTION
**Pull request by riidefi**
`/i [item] [qty|0=STICKY]`

`/i`: Clear the player's item
`/i 6`: Give the player 1 bob-omb
`/i 5 2`: Give the player 2 mushrooms
`/i 5 -2`: Give the player permanent 2 mushrooms

### CODE REVIEW

File: payload/egg/core/eggSystem.c [hunk 1]
```diff
@@ -78,6 +76,81 @@ static void DolphinTest() {
...
+static void TrySetItem(s32 myPlayerId, s32 item, s32 qty) {
+    if (s_itemDirector != NULL) {
+        s_itemDirector->mKartItems[myPlayerId].mCurrentItemKind = item;
+        s_itemDirector->mKartItems[myPlayerId].mCurrentItemQty = qty;
+    }
+}
+
+// Matches "/command", "/command arg", but not "/command2"
+#define STRING_STARTSWITH_COMMAND(s, cmd) \
+    (!strcmp(s, cmd) || !strncmp(tmp, cmd " ", strlen(cmd " ")))
```
Comment (stblr) [inline payload/egg/core/eggSystem.c:113]: This should be a function.

File: payload/egg/core/eggSystem.c [hunk 2]
```diff
@@ -78,6 +76,81 @@ static void DolphinTest() {
     IOSDolphin_Close(dolphin);
 }
 
+static s32 GetMyPlayerID(void) {
+    if (s_raceConfig == NULL) {
+        OSReport("&cNot in a race.\n");
```
Comment (stblr) [inline payload/egg/core/eggSystem.c:81]: Certainly out of scope for this PR but we will want to strip the color codes when not printing to the screen.

**Review by stblr — CHANGES_REQUESTED** [review]

File: payload/egg/core/eggSystem.c (hunk 2 above)
Comment (riidefi) [reply payload/egg/core/eggSystem.c:81]: Agreed

File: payload/egg/core/eggSystem.c (hunk 1 above)
Comment (riidefi) [reply payload/egg/core/eggSystem.c]: Fixed

**Review by stblr — APPROVED** [review]

**Pull request merged**